app/app.html

```diff
@@ -0,0 +1,17 @@
+<ion-menu id="leftMenu" [content]="content">
```

Comment:
Here we are setting the content property of the menu to content. Keep in mind that “content” here is an expression, not a string. We’re not setting the content property to a string “content” we are setting it to the variable “content”